src/components/RegisterPage.tsx

```diff
@@ -114,6 +118,9 @@ const RegisterPage = () => {
       />
     );
   };
+  if (userId) {
+    return <Navigate to="/recipes" />;
```

Comment:
fyi: this approach or programmatically [useNavigate hook] - sometimes code is more clean with the 2nd approach - not in this case; just notifying